bounties_api/analytics/management/commands/timeline_generator.py

```diff
@@ -260,10 +267,41 @@ def handle(self, *args, **options):
                 # and provides a better way to expose by day or by hour in case of been needed
                 for day in days:
                     bounty_day = generate_timeline(day_bounds(day), schema=schema)
-                    bounty_points = BountiesTimeline.objects.filter(date=day.date(), schema=schema)
+                    bounty_points = BountiesTimeline.objects.filter(date=day.date(), schema=schema, is_week=False)
 
                     if bounty_points.exists():
                         bounty_point = bounty_points.first()
                         bounty_day.id = bounty_point.id
 
                     bounty_day.save()
+
+        for schema in schemas:
+            needs_genesis = not BountiesTimeline.objects.filter(schema=schema, is_week=True).count()
+
+            first_date = BountyState.objects.first()
+            last_date = BountyState.objects.last()
```

Comment:
Another issue, and this was already an issue, is the end date is based on the latest bounty. It should be based on the current UTC time. Otherwise, this means a date has no stamp or record until a bounty action happens in that date. Makes sense? This is an issue.  And this was also already an issue.